src/ExpoMessage.php

```diff
@@ -219,15 +221,16 @@ public function priority(string $priority)
     public function toArray()
     {
         $message = [
-            'title'     =>  $this->title,
-            'body'      =>  $this->body,
-            'sound'     =>  $this->sound,
-            'badge'     =>  $this->badge,
-            'ttl'       =>  $this->ttl,
-            'data'      =>  $this->jsonData,
-            'priority'  =>  $this->priority,
+            'title' => $this->title,
+            'body' => $this->body,
+            'sound' => $this->sound,
+            'badge' => $this->badge,
+            'ttl' => $this->ttl,
+            'data' => $this->jsonData,
+            'priority' => $this->priority,
+            'interruptionLevel' => $this->interruptionLevel->value,
```

Comment:
it won't work for the `null` value, please add this field only in case the `interruptionLevel` is not empty